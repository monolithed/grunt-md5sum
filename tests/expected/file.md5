/fixtures/file.css  5fcf8400077ea9f88d2039c81eabe63d
/fixtures/file.js  8515d9ad279c8115b48fc796a9b1a3a3
