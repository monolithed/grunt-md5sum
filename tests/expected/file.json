[
	{
		"file": "tests/fixtures/file.css",
		"hash": "5fcf8400077ea9f88d2039c81eabe63d"
	},
	{
		"file": "tests/fixtures/file.js",
		"hash": "8515d9ad279c8115b48fc796a9b1a3a3"
	}
]